CHANGELOG.md

```diff
@@ -43,6 +43,7 @@
     - save_as_table
 - Added support for snow:// URLs to `snowflake.snowpark.Session.file.get` and `snowflake.snowpark.Session.file.get_stream`
 - UDAF client support is ready for public preview. Please stay tuned for the Snowflake announcement of UDAF public preview.
+- Added new API ( snowflake.snowpark.Session.lineage.trace ) to explore data lineage of snowfake objects.
```

Comment:
1. You may need to merge the main branch and move this to unreleased. 
2. I suggest `Added snowflake.snowpark.Session.lineage.trace ...`